src/ludamus/templates/panel/timetable.html

```
@@ -31,6 +31,60 @@
+<div class="px-3 sm:px-4 pt-3 flex justify-end gap-3 flex-wrap items-center">
+    <div class="flex items-center gap-1">
+        <a href="{% url 'panel:timetable-print' slug=current_event.slug %}"
+           target="_blank"
+           rel="noopener"
+           class="text-sm px-3 py-1.5 border border-border rounded-lg bg-bg-secondary text-foreground hover:bg-bg-tertiary transition">
+            {% translate "Print timetable" %}
+        </a>
+        {% if print_venues %}
+            <select onchange="if(this.value){window.open(this.value,'_blank','noopener');this.selectedIndex=0;}"
+                    aria-label="{% translate "Print timetable for a venue or area" %}"
+                    class="text-sm border border-border rounded-lg px-2 py-1.5 bg-bg-secondary text-foreground focus:outline-none focus:ring-2 focus:ring-primary">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)